MdePkg/Test/Mock/Include/GoogleTest/Ppi/MockReadOnlyVariable2.h

```diff
@@ -0,0 +1,43 @@
+/** @file MockReadOnlyVariable2.h
+  This file declares a mock of Read-only Variable Service2 PPI.
+  This ppi permits read-only access to the UEFI variable store during the PEI phase.
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __MOCK_PEI_READ_ONLY_VARIABLE2_PPI_H__

```

Comment:
```suggestion
#ifndef MOCK_PEI_READ_ONLY_VARIABLE2_PPI_H__

```

include guards with _ are reserved for the compiler: https://tianocore-docs.github.io/edk2-CCodingStandardsSpecification/release-2.20/5_source_files/53_include_files.html#53-include-files

Reply:
Ahh I see, no problem! Any reason Uncrustify doesn't notice it? I'm guessing because there are exceptions? I noticed the underscores are there in PeiServicesLib.h and in PPI header files.

Reply:
I don't think uncrustify can make a distinction between an include file that is compiler specific and one that we generate.

Reply:
This convention started after uncrustify, but uncrustify is more about code structure than naming and textual conventions.

ECC probably checks for this, but we don't run that. Also, sometimes these things change and only new code gets flagged (by tools like ECC).